src/main/java/uk/gov/justice/digital/hmpps/whereabouts/dto/AppointmentDtos.kt

```diff
@@ -156,8 +156,13 @@ data class VideoLinkAppointmentDto(
   val bookingId: Long,
   @ApiModelProperty(value = "Appointment id, maps to nomis event id", example = "1")
   val appointmentId: Long,
-  @ApiModelProperty(value = "The location of the court that requires the appointment", example = "York Crown Court")
+
+  @ApiModelProperty(value = "The name of the court that requires the appointment", example = "York Crown Court")
   val court: String,
```

Comment:
I think this doc needs to match that used in VLBResponse (it has a note of "At least one of court or courtId will be returned").

Reply:
Done